tx_bucket.go

```diff
@@ -20,20 +20,16 @@ func (tx *Tx) IterateBuckets(ds uint16, pattern string, f func(bucket string) bo
 		return err
 	}
 
-	handle := func(bucket string) error {
-		if end, err := MatchForRange(pattern, bucket, f); end || err != nil {
-			return err
-		}
-		return nil
-	}
 	var err error
```

Comment:
I thinks this err could be removed and just return nil in the end.

Reply:
Okay, I've fixed it.